src/maidsafe/launcher/ui/controllers/account_handler_controller.h

```diff
@@ -43,33 +49,38 @@ class AccountHandlerController : public QObject {
     CreateAccountView,
   };
 
-  AccountHandlerController(helpers::MainWindow& main_window, QObject* parent);
+  AccountHandlerController(MainWindow& main_window, QObject* parent);
+  ~AccountHandlerController() override;
+  AccountHandlerController(AccountHandlerController&&) = delete;
+  AccountHandlerController(const AccountHandlerController&) = delete;
+  AccountHandlerController& operator=(AccountHandlerController&&) = delete;
+  AccountHandlerController& operator=(const AccountHandlerController&) = delete;
 
   AccountHandlingViews currentView() const;
-  void setCurrentView(const AccountHandlingViews new_current_view);
-  Q_SIGNAL void currentViewChanged(AccountHandlingViews arg);
+  void SetCurrentView(const AccountHandlingViews new_current_view);
```

Comment:
No, because the type here is enum (with int as underlying_type). I think it is wasteful to have references for built-in primitives because copy should be more efficient for them. But if it's otherwise (or in coding convention) then I'll change it.